app/uk/gov/hmrc/merchandiseinbaggage/config/AppConfig.scala

```diff
@@ -16,51 +16,44 @@
 
 package uk.gov.hmrc.merchandiseinbaggage.config
 
+import com.google.inject.Inject
+import play.api.Configuration
+import uk.gov.hmrc.merchandiseinbaggage.config.AppConfig._
 import javax.inject.Singleton
```

Comment:
Can just use this instead for Inject: `import javax.inject.{Inject, Singleton}` and delete the one on line 19, thanks.